samples/constitutions/sandbox/sandbox/resolve.js

```diff
@@ -0,0 +1,7 @@
+export function resolve(proposal, proposerId, votes) {
+  // The constitution in a real CCF application deployment would at least
+  // count votes and compare to a threshold of members, but in the sandbox sample,
```

Comment:
```suggestion
  // count votes and compare to a threshold of active members, but in the sandbox sample,
```

Reply:
This file shouldn't exist (it's a new `sandbox/sandbox` file, I don't know how I did that), but I'll make this change in the correct copy.